webapp/src/pages/UserPage/UserPage.tsx

```diff
@@ -0,0 +1,85 @@
+import './UserPage.css';
+import {
+  Button,
+  ButtonVariant,
+  Header,
+  IconVariant,
+  LargeAvatar,
+  Row,
+  Text,
+  TextColor,
+  TextVariant,
+  TextWeight,
+} from '../../shared/components';
+import { USER_URL, USERS_EP_URL, WILDCARD_AVATAR_URL } from '../../shared/urls';
+import { instanceOfUser, User } from '../../shared/types';
+import { useData } from '../../shared/hooks/UseData';
+import { goBack, logout } from '../../shared/callbacks';
+import { useEffect, useState } from 'react';
+
+export default function UserPage() {
+  const [me, setMe] = useState<User>({
+    username: '',
+    email: '',
+    avatarId: null,
+    id: '',
+    createdAt: new Date(Date.now()),
+  });
```

Comment:
thought: In the first render of this page, the user has empty info. We could assign the `me` state to `null` and return the `user-page` or a `Loading` component (for now, a loading… text, until we create this component, it could be a spinner or something like that)

```typescript
  const [me, setMe] = useState<User | null>(null);

  return me === null ? (
    <div>Loading...</div>
  ) : (
    <div className="user-page">
 ...
    </div>
  );
```

Reply:
done